app/views/browse/routes.html

```diff
@@ -24,7 +24,22 @@
         <div class="container-fluid">
           <div class="row">
             <div class="col-md-12 gutter-top">
-              <table class="table table-bordered table-hover table-mobile">
+              <div ng-if="(routes | hashSize) == 0" class="blank-slate-pf">
```

Comment:
You may be able to do `ng-if="!(routes | hashSize)"` and `ng-if="routes | hashSize"` and drop the `== 0`.